workflow-dispatch/action.yml

```diff
@@ -0,0 +1,25 @@
+name: Workflow Dispatch
+description: Triggers a workflow with the workflow_dispatch event.
+inputs:
+  owner:
+    description: "Name of the repository owner in which the workflow resides. Defaults to the one who owns the workflow in which this action is running."
+    required: false
+  repo:
+    description: "Name of the repository in which the workflow resides. Defaults to the one in which the workflow currently running resides."
+    required: false
+  token:
+    description: "Github token. Cannot be the default github token provided in the worflow run."
```

Comment:
Instead of saying what it cannot be, maybe it would be better to specify what it must be?